sdk/servicebus/service-bus/test/internal/unit/streamingReceiver.spec.ts

```diff
@@ -190,7 +176,7 @@ describe("StreamingReceiver unit tests", () => {
 
       // closeLink is called on cleanup when we fail to add credits (which we would because our receiver
       // was suspended, which will cause us to fail to add credits)
-      assert.isTrue(closeLinkSpy.called);
+      assert.isTrue(closeLinkSpy.mock.calls.length > 0);
```

Comment:
Could we use `expect(closeLinkSpy).toHaveBeenCalled();`?

Reply:
updated.